java/arcs/core/data/testdata/WriterReaderExample.arcs

```diff
@@ -0,0 +1,21 @@
+particle Reader
+  data: reads Thing {name: Text}
+
+particle Writer
+   data: writes Thing {name: Text}
+
+@trigger
+  launch startup
+  arcId writingArcId
+recipe WritingRecipe
```

Comment:
If you don't like the "Recipe" in the name, feel free to rename it. it can be "Ingestion", and the reading one could be "Consumption" or similar.